### ISSUE TITLE
contentFields section tweaks

### ISSUE THREAD

**Issue by sebastianlenton** [area17/twill#117]
Hi,

I have a contentFields section that looks like this (see below), the only field being the block editor. There are two things i want to do:

1. How do I change the "Content" title?
2. How do I get rid of the grey line above the block editor button?

Thanks!

![screen shot 2018-12-13 at 15 53 59](https://user-images.githubusercontent.com/1895476/49950253-5adef480-feef-11e8-8b9e-c1fb8ed7b62f.png)

**Comment by ifox** [comment]
Hi Seb,

1. Use `contentFieldsetLabel` when extending the form layout:

```
@extends('twill::layouts.form', [
    'contentFieldsetLabel' => 'Custom title for the default form box',
])
```
You can also disable it entirely and use your own fieldsets using `'disableContentFieldset' => true`.

2. Use `'withoutSeparator' => true` in your `@formField`.

**Comment by ifox** [comment]
Oh I'm sorry this is actually new and not part of a new release yet. You coulf use the 1.2.x-dev version in the meantime.

**Comment by sebastianlenton** [comment]
No biggie- as said I managed to sort anyway, by renaming the contentFields section. Just wondering.

Thanks as ever!

**Comment by zipavlin** [comment]
> I couldn't get `'disableContentFieldset' => true` to do anything though- is it done as follows, or have I put it in the wrong place?

You put it in the wrong place :)
It should be in formField:
```
@formField('block_editor', [
    'withoutSeparator' => true,
    'blocks' => ...
])
```

**Comment by ifox** [comment]
Just to clarify guys, `disableContentFieldset` goes into the `@extends` directive, but this is not released yet and can be tested on the `1.2` branch.

**Comment by Craigeniah** [comment]
This issue was very helpful, couldn't figure out why there was a horizontal rule ( <hr> tag ) appearing at the top of our edit "page".
